### DISCUSSION TITLE
Beretta M9 and Glock 19 are the exact same gun

### DISCUSSION THREAD

**Discussion by AriaMoradi** [cataclysmbn/Cataclysm-BN#4401] [General]
### Is your feature request related to a problem? Please describe.

According to the design doc this should be cleaned up

### Describe the solution you'd like

remove one of the guns(and increase the spawn rate of the other) or make one different enough to the other.

**Comment by NerdPunkFu** [comment]
One issue to note here is that M9 magazines are used by Cx4s and 90-twos as well. Outright removal of the M9 should address this as well.

**Comment by Lamandus** [comment]
the gun itself is not the most important part, the magazines are. Like NFP already said, the M9 has cross compatibility. Not sure if the Glock 19 is using special magazines or the same as the Glock 17 and or Glock 21.

**Comment by ghost** [comment]
Our design docs are from DDA era and likely haven't been updated in a good 5 years, so they should not be taken seriously unless there's a general consensus among current maintainers on a rewrite. We don't have a reason to follow DDA rules on allowed items. 

Glocks and Beretta's are different companies and the M9 is the standard service pistol in the Army (so it's going to be fairly common), they take different magazines which in turn are used by different weapons, and would have different spawn locations. Handguns of the same caliber in general are all the same but there isn't really a bloat issue in the handguns. This sounds like territory for generic guns to solve.

**Comment by ghost** [comment]
Coming back to this since notifications are fun, barrel length differs IRL between these two and likely would muzzle velocity (thus damage), probably dispersion, handling, size, weight. We have a half dozen or dozen ways to make firearms slightly different from one another.
